fitInTests/ScenarioTest.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  ScenarioTest.swift
+//  fitInTests
+//
+//  Created by Kevin C on 2017-11-03.
+//  Copyright © 2017 AJ productions. All rights reserved.
+//
+
+import XCTest
+@testable import fitIn
+
+class ScenarioTest: XCTestCase {
+    //create a scenario we will fudge with
+    let testScenario = Scenario()
```

Comment:
Create a new scenario for each test case. In these tests all we check for is types but if there are any other types of tests then they will be order dependent on which tests run first.